src/_pytest/_io/terminalwriter.py

```diff
@@ -32,7 +32,7 @@ def should_do_markup(file: TextIO) -> bool:
         and file.isatty()
         and os.environ.get("TERM") != "dumb"
         and not (sys.platform.startswith("java") and os._name == "nt")
-    )
+    ) or os.environ.get("GITHUB_ACTIONS") == "true"
```

Comment:
I think it will be a little clearer as a separate `if` before the `return` (can leave as is if you prefer)

Reply:
Sounds like a good idea, since we then handle all the special cases before the "ordinary" one.

Reply:
Moved!